src/main/java/hudson/plugins/parameterizedtrigger/BuildTriggerConfig.java

```diff
@@ -109,24 +114,36 @@ public boolean getTriggerWithNoParameters() {
 
     /**
      * @deprecated
-     *      Use {@link #getProjectList(ItemGroup, EnvVars)}
+     *      Use {@link #getProjectListGeneral(ItemGroup, EnvVars)}
      */
     public List<AbstractProject> getProjectList(EnvVars env) {
-        return getProjectList(null,env);
+        return getProjectList(null, env);
     }
 
     /**
+     * Deprecated: get all projects that are AbstractProject instances
      * @param env Environment variables from which to expand project names; Might be {@code null}.
      * @param context
      *      The container with which to resolve relative project names.
      */
+    @Deprecated // Prefer getProjectListGeneral since it can return implementations of the more general Job class
```

Comment:
I would add ```@deprecated``` javadoc also (more visible from IDEs when developing and Javadoc pages itself)